bigquery/tests/unit/test_client.py

```diff
@@ -272,6 +272,32 @@ def test_dataset_with_default_project(self):
         self.assertEqual(dataset.dataset_id, DATASET)
         self.assertEqual(dataset.project_id, PROJECT)
 
+    def test_get_dataset(self):
+        project = 'PROJECT'
+        dataset_id = 'dataset_id'
+        path = 'projects/%s/datasets/%s' % (project, dataset_id)
+        creds = _make_credentials()
+        http = object()
+        client = self._make_one(project=project, credentials=creds, _http=http)
+        DS_ID = '%s:%s' % (project, dataset_id)
```

Comment:
Since you don't use this variable again, maybe just put the % expression in the resource dict?

Reply:
Made change with https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3973/commits/811c9b03d5825d514347e16ff43d8e1948906a2f